webrtc.html

```diff
@@ -7484,14 +7485,15 @@ <h2>
         possible designs have been discussed in <a href=
         "https://github.com/w3c/webrtc-pc/issues/1283">GitHub issue 1283</a>.
       </p>
-      <p>
-        When video is rescaled, for example for certain combinations of width
-        or height and {{RTCRtpEncodingParameters/ scaleResolutionDownBy}}
-        values, situations when the resulting width or height is not an integer
-        may occur. In such situations the user agent MUST use <a href=
-        "https://tc39.github.io/ecma262/#eqn-floor">the integer part of the
-        result</a>. What to transmit if the integer part of the scaled width or
-        height is zero is implementation-specific.
+      <p id="rtp-video-rescaled">
+        Whenever video is rescaled as a result of {{RTCRtpEncodingParameters/scaleResolutionDownBy}},
+        situations when the resulting width or height is not an integer
+        may occur. The user agent MUST NOT transmit video larger than
+        <a href="https://tc39.github.io/ecma262/#eqn-floor">the integer part</a>
+        of the scaled width and height from
+        {{RTCRtpEncodingParameters/scaleResolutionDownBy}}. What to transmit if
+        the integer part of the scaled width or height is zero is
+        [=implementation-defined=].
```

Comment:
This needs to be a bit broader -- in case of resolution alignment it is not just the integer part of the scaled dimensions being zero that is implementation-defined.

Also [some codecs impose a minimum resolution](https://developer.android.com/reference/android/media/MediaCodecInfo.VideoCapabilities#getSupportedWidths()) that could impact this.

Reply:
This particular paragraph now only discusses scaleResolutionDownBy, so hopefully it is clear that this sentence only speaks to that: basically a clarification of the sentence that precedes it, which otherwise would less unclear.

I don't think it is claiming this to be the _only_ implementation-defined behavior.

Regarding minimum codec resolution, that is a good point. I've added that. PTAL.